blog_remake/db.json

```diff
@@ -0,0 +1,124 @@
+        "Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia natus, sit aut tempore delectus earum numquam placeat tempora modi sed facere fuga perferendis aliquam quidem culpa praesentium? Ad vitae laborum cum tenetur doloremque soluta saepe explicabo, provident corporis, cumque a quae eligendi repudiandae quo facilis ducimus hic nesciunt voluptate neque corrupti. Qui provident ipsum beatae unde voluptatibus laudantium ipsa? Eaque!",
+        "Lorem ipsum dolor sit amet consectetur, adipisicing elit. Repudiandae mollitia tempora reiciendis odio vero autem voluptatem quia sed! Maiores, voluptate ab corrupti sequi recusandae perspiciatis at tempora ut laudantium sint natus cupiditate accusantium aperiam deserunt officia laboriosam quae, eum repellat? Molestiae sint voluptate amet similique obcaecati animi at accusamus odio? Impedit cumque esse, voluptas at provident est. Suscipit praesentium consequatur impedit laborum veniam voluptatem, maiores nam dolorem quidem expedita nesciunt corrupti iusto ipsa eos asperiores assumenda maxime magnam libero iste. Debitis repudiandae sed necessitatibus reprehenderit quisquam illo sapiente doloremque unde, voluptatum commodi assumenda amet ab perspiciatis magnam dignissimos exercitationem ex blanditiis nemo, nihil at iste non. Autem cumque reiciendis itaque natus placeat soluta, laboriosam repellendus sapiente! Sint nisi aspernatur eaque?",
+        "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Saepe quaerat ratione quis impedit! Exercitationem, ipsa. Placeat quod voluptate hic nam incidunt temporibus quis blanditiis eos architecto iste commodi, est ex voluptatum veniam, ut nisi obcaecati impedit debitis! Fugiat consequatur dolorum, possimus quo magni aspernatur at quia repudiandae animi nam debitis.",
+        "Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut earum praesentium temporibus tempora? Magni, molestiae. Ab deleniti, totam a, amet pariatur doloremque iusto consequatur ducimus assumenda libero architecto id ullam dolor, ipsa voluptate laboriosam enim sunt magni magnam vitae molestiae! Soluta facilis repellendus explicabo id dolore ex nam quis nostrum."
+      ],
+      "quote": "Life is like riding a bicycle, to keep your balance you must keep moving",
+      "linkReadMore": "#articles/tlod1718"
+    },
+    {
+      "id": "5",
```

Comment:
Was it a choice to create the IDs as strings?

Reply:
i did not realize, i think it's from prettier. I will fix this in a new commit.